src/lib/features/project/project-service.ts

```diff
@@ -304,6 +304,30 @@ export default class ProjectService {
         return featureNaming;
     };
 
+    private validateLinkTemplates = (
+        linkTemplates: IProject['linkTemplates'] | unknown,
```

Comment:
I'd only expect unknown at the boundary of the system before openapi validation. I'm surprised by seeing unknown at the service level